sdks/python/apache_beam/dataframe/frames.py

```diff
@@ -922,6 +922,19 @@ def mask(self, cond, **kwargs):
     """mask is not parallelizable when ``errors="ignore"`` is specified."""
     return self.where(~cond, **kwargs)
 
+  @frame_base.with_docs_from(pd.DataFrame)
+  @frame_base.args_to_kwargs(pd.DataFrame)
+  @frame_base.populate_defaults(pd.DataFrame)
+  def truncate(self, before, after, axis):
+    return frame_base.DeferredFrame.wrap(
+        expressions.ComputedExpression(
+            'truncate',
+            lambda df: df.truncate(before=before, after=after, axis=axis),
+            [self._expr],
+            requires_partition_by=partitionings.Arbitrary(),
+            preserves_partition_by=partitionings.Arbitrary()
+            if axis in (1, 'columns') else partitionings.Singleton()))
```

Comment:
I don't think this `if` should be necessary, did you have issues without it?

Reply:
Oh sorry I didn't have issues, I was testing and forgot to change it.